<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenewWidget</class>
 <widget class="QWidget" name="RenewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LabelSelect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>功能选择：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="BtnChange">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>修改数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnDelete">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>删除数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnAdd">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>增加数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnSource">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>更新源文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="DeleteWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>591</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QLabel" name="LabelDelete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正姚体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>要删除信息的LinkID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineDelete">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnForDelete">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelSuccess">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>删除成功</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelFail">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>没有找到</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LabelWelcome">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>请选择要进行的操作！</string>
   </property>
  </widget>
  <widget class="GetData" name="NewWidget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>605</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="ChangeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>591</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>方正姚体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>要修改的数据的LinkID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="BtnChang">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GetData</class>
   <extends>QWidget</extends>
   <header location="global">getdata.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
